packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -5,6 +5,15 @@ const defaultOptions = require('./utils/default-options');
 
 const isProd = process.env.NODE_ENV === 'production';
 
+const wrapESMPlugin = (name) =>
```

Comment:
This is great!  Would it be horrible scope creep to do this with the other remark / rehype plugins in one go and remove all those pesky manual renovate version constraints?

Reply:
I can give it a try. Hopefully it will not create huge breaking changes

Reply:
in terms of what's directly used in `gatsby-config.js`
<img width="531" alt="Screenshot 2023-01-23 at 15 33 49" src="https://user-images.githubusercontent.com/106664971/214065738-9a1afe7b-57d3-42b2-b9b4-a1f96d8618cc.png">
I managed to upgrade all the remark/rehype plugins out of the box, except for `remark-gfm` which seems to be using a different pattern regarding what the default function returns. It might be possible upgrate it, but it's not effortless. Probably it's worth to upgrate to gatsby 5 which should support ES6.

Reply:
nice!  We'll get that renovate configuration slimmed down piece by piece.